modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -447,6 +454,10 @@ else if (StringUtil.equals(
 						objectValidationRuleEngine.getKey(),
 						ObjectValidationRuleConstants.ENGINE_TYPE_DDM)) {
 
+				_addValues(
```

Comment:
I think we can make this method more specific to know which values are being added.
What do you think about `addRelatedObjectEntryValues()` ? Would you happen to have a suggestion?

Reply:
I agree! Done!